src/content/general/management-clusters/index.md

```diff
@@ -25,21 +25,36 @@ As well as operators, our management clusters run supporting components such as
 
-The components are deployed as App custom resources using our App Platform. Changes are tested in a number of internal management clusters before being deployed to all management clusters.
+These management cluster-only components are tested on our internal management clusters before releases are cut.
+
+We plan on replacing draughtsman in Q2 2021, with a new component that more natively synchronises app collections to a management cluster, mostly to improve the scalability of the system.
 
 ## Configuration changes
 
-From time to time, we also need to update the configuration for management clusters. These changes are also tested on our internal management clusters before being applied.
+Periodically, we need to update the configuration for management clusters.
+
+This configuration is currently held in a ConfigMap and a Secret stored on the Management Cluster. When updates are required, these resources are updated manually using our current tooling, from our central configuration repository.
+
+These configuration files are watched by the App Platform, and any necessary applications are reconfigured on changes.
+
+Changes to configuration are tested on our internal management clusters before being applied.
+
+This configuration system is being updated in Q1 2021, with a system based on the new [config-controller](https://github.com/giantswarm/config-controller/). This will remove the need for manual updates of configuration, remove the shared ConfigMap and Secret, and allow for versioning of configuration. Overall, aiming to further increase the safety, reliability, and scalability of configuration changes.
 
 ## Infrastructure upgrades
 
-The infrastructure including the management cluster nodes and supporting resources like bastion hosts are managed by our SRE team using Terraform.
+The management cluster infrastructure, such as the Kubernetes cluster itself, or other supporting resources such as bastion hosts, are managed using Terraform.
+
+Updates to our Terraform playbooks are rolled out by CI as needed, with changes to our Terraform playbooks being tested on internal management clusters before being deployed.
 
-However, we plan to move more of this provisioning to operators, reusing the components that manage our workload clusters where possible. As they continuously reconcile the cluster to its desired state, operators are more dynamic than Terraform and other tools such as Kops.
+In 2021, we plan to start reusing our own workload clusters as the clusters that base management clusters, to both avoid having to maintain two methods of provisioning clusters, but also to gain the reliability benefits of our workload clusters.
```

Comment:
The bit about operators being more dynamic was taken from https://www.giantswarm.io/management-cluster

@teemow asked if we could consolidate the docs with the website page.
https://github.com/giantswarm/giantswarm/issues/15700#issuecomment-780029135

Reply:
Fine to drop it from my POV if you think it isn't needed.

Reply:
i'll take this into an issue in biscuit, because we can be a bit more aligned here, sure

Reply:
(see https://github.com/orgs/giantswarm/projects/125#card-55271871)